src/app/demo/templates/templates.component.html

```diff
@@ -114,6 +114,9 @@ <h2 ngNonBindable>Event binding: (event)</h2>
       [readonly]="true"
       type="html">
     </slides-code-editor>
+    <div class="info">
+      Parentheses <b>(click)</b> are used for event binding, but "on-" can also be used <b> on-click</b>
```

Comment:
ok, I'll leave the "on-event" one as is, since it's in the info section, but I'll move the "bind-attribute" one to the additional info section.